src/components/widgets/sales-cycle/sales-cycle.directive.coffee

```diff
@@ -45,17 +47,23 @@ module.controller('WidgetSalesCycleCtrl', ($scope, $q, ChartFormatterSvc, $filte
     if error.code == 404
       $scope.isDataFound = false
 
+  $scope.getAccountColor = (elem) ->
+    ChartFormatterSvc.getColor(_.indexOf($scope.pieData, elem)) if $scope.isDataFound
+
+  getName = (name) ->
+    if name?
+      return name.replace(/_/g, " ")
 
   # Chart formating function
   # --------------------------------------
   $scope.drawTrigger = $q.defer()
   w.format = ->
     if $scope.isDataFound
-      pieData = _.compact _.map $scope.statusOptions, (statusOption) ->
+      $scope.pieData = _.compact _.map $scope.statusOptions, (statusOption) ->
```

Comment:
why adding this `pieData` variable to the `$scope` there? do you need it in the template?